LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright  (c) 2017 AgileBits Inc.
+Copyright &copy; 2017-2018 AgileBits Inc.
```

Comment:
The `(c)` notation is pretty standard in text licenses. Even though it's markdown, it's nice to be able to read it as plain text.